configuration/pih/pih-config-haiti.json

```diff
@@ -5,39 +5,6 @@
 
     "country": "HAITI",
 
-    "components": [
```

Comment:
Not sure why we had this, but there really isn't a need to set components in this top-level "Haiti" file, as we will aways be using it in conjunction with a country file like "pih-config-haiti-cange.json" or "pih-config-haiti-hiv.json"